### ISSUE TITLE
ViewToggle: axe-core validation fails

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1680]
getting an error that `aria-selected` is not a valid attribute on a `button` element. I believe we need to do some research on the appropriate pattern.
